public/index.html

```diff
@@ -25,52 +25,144 @@
       rel="stylesheet"
       type="text/css"
     />
+    <link
+      href="https://cdn.materialdesignicons.com/5.0.45/css/materialdesignicons.min.css"
+      rel="stylesheet"
+      type="text/css"
+    />
 
     <!-- Load Skeleton CSS -->
```

Comment:
Nit: Change to "Load CSS files" or something similar

Reply:
Good call, I've rectified it.